hermes-api/src/main/java/pl/allegro/tech/hermes/api/SubscriptionHealthProblem.java

```diff
@@ -0,0 +1,116 @@
+package pl.allegro.tech.hermes.api;
+
+public enum SubscriptionHealthProblem {
```

Comment:
this class is misplaced - API should contain only simple enum which is used in API. hide calculations in some better place, this way you wont have to create some strange inner classes.